core/schema/platform.graphqls

```diff
@@ -1,3 +1,7 @@
+"""
+The platform config OS and architecture in a Container.
+The format is <os>/<platform>/<version> (e.g. darwin/arm64/v7, windows/amd64, linux/arm64).
```

Comment:
@TomChv Your deployment errors are coming from this part. `<os>/<platform>/<version>`.
When generating docs reference the typedoc plugin is considering those as React components (weird...)
So a workaround would be to use a different syntax.
eg: 
`os/platform/version`
`{os}/{platform}/{version}`
...

Reply:
Thanks for the explanation! I'll fix that